### PR TITLE
Update `oncoprint-landscape` module documentation and oncoprints

### PR DESCRIPTION
**Pull request by cbethell**
### Purpose/implementation Section

The purpose of this PR is to include the recurrent genes list generated in PR #686 in the oncoprint plots and to update the `oncoprint-landscape` module documentation .

#### What scientific question is your analysis addressing?

This PR is working to reflect the genes with recurrent CNVs on the oncoprint landscape PNGs, as well as update the `oncoprint-landscape/README.md` file.

#### What was your approach?

The approach in this PR, as mentioned on issue #706 was to:

- Write a script that takes in the two gene lists (`analyses/oncoprint-landscape/results/consensus_seg_focal_cn_recurrent_genes.tsv` and `analyses/interaction-plots/results/gene_disease_top50.tsv`) and combines them (for use in the oncoprint plotting)
- Update the `01-plot-oncoprint.R` and `run-oncoprint.sh` scripts to use the concatenated list of genes
- Update the figure found at `figures/pngs/fig3-oncoprint-landscape.png`
- Update the `oncoprint-landscape/README.md file` to include the origin of the genes of interest list(s) given to the `oncoplot` function via the module shell script and the structure of the directory

#### What GitHub issue does your pull request address?

#706

#### Which areas should receive a particularly close look?

- How do the plots look with the top ~20 recurrent CN genes (from `focal-cn-file-preparation` directory) and the top 50 mutated genes (from `interaction-plots` directory)? Maybe I should take only the top 20 mutated genes to avoid clutter or make the size of the gene labels a bit smaller on the goi oncoprints?
- Does the documentation need any clarifications?

#### Is there anything that you want to discuss further?

Note that I also removed the files stored in `oncoprint-landscape/driver-lists` as they are no longer being used in this module.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?

Yes.

#### What types of results are included (e.g., table, figure)?

The PNG files in `oncoprint-landscape/plots` have been updated, as well as `figures/pngs/fig3-oncoprint-landscape.png`

The concatenated gene lists can also be found in `oncoprint-landscape/results/combined_goi_list.tsv`

#### What is your summary of the results?

The final plots seem to show a clear split between the genes provided via the list produced in the `interaction-plots` directory versus the list of genes provided via the `focal-cn-file-preparation` directory. This may change the way we want to decide which genes are represented on the oncoprint plots.

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/oncoprint-landscape/README.md [hunk 1]
```diff
@@ -1,30 +1,49 @@
...
   * Filtering via an [independent specimen file](https://alexslemonade.github.io/OpenPBTA-manuscript/#selection-of-independent-samples) is optional, but highly recommended.
-* `01-plot-oncoprint.R` takes the files from above and optionally a gene list and creates an oncoprint.
+* `01-plot-oncoprint.R` takes the files from above and optionally a list of top mutated genes (generated in `analyses/interaction-plots/scripts/01-disease-specimen-lists.R`, top genes with recurrent CNVs (generated in `analyses/focal-cn-file-preparation/06-find-recurrent-calls.Rmd`, or a concatenated list of these genes (generated in this module's `util/concatenate-gene-lists.R` script).
+This script generates an oncoprint displaying the landscape across PBTA given
+the relevant metadata.
+* `concatenate-gene-lists.R` takes two separate gene lists as input and combines them into a list with one column for use in the oncoprint plotting.
+The output of this script is a TSV file with the combined list of genes, stored in this module's `results` directory.
+This script is found in this module's `util` directory.
+
+### Folder Structure
```
Comment (jaclyn-taroni) [inline analyses/oncoprint-landscape/README.md:30]: You added back the driver lists with https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/711/commits/cb210ea1fbd15c00d152dc2f305e9f676d2191f5. They should be represented here and also the source should be documented as stated in #706

**Comment by jaclyn-taroni** [comment]
> How do the plots look with the top ~20 recurrent CN genes (from focal-cn-file-preparation directory) and the top 50 mutated genes (from interaction-plots directory)? Maybe I should take only the top 20 mutated genes to avoid clutter or make the size of the gene labels a bit smaller on the goi oncoprints?

There's a more general problem with selecting the top 20 genes from the CNV data across the entire cohort from a "is this likely to be worth talking about/displaying" perspective. We know that there are probably false positives in that data: https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/fd0d6f02516e21644446b3392dd0912b4369a22e/figures/pngs/cn_status_heatmap.png

**Comment by jaclyn-taroni** [comment]
Can you tell me about the design decision for having a separate script that generates the concatenated gene lists?

**Comment by cbethell** [comment]
> Can you tell me about the design decision for having a separate script that generates the concatenated gene lists?

The decision to have a separate script to generate the concatenated gene lists was made in the event that one does not need to use the script because there is only one desired gene list to supply to the `oncoplot` function. In other words, it is meant to be a script that can be used if needed, but does not have to be.

**Comment by jaclyn-taroni** [comment]
> The decision to have a separate script to generate the concatenated gene lists was made in the event that one does not need to use the script because there is only one desired gene list to supply to the oncoplot function. In other words, it is meant to be a script that can be used if needed, but does not have to be.

I would argue you could accomplish the same thing altering `01-plot-oncoprint.R` to accept a comma- or space-separated list of files so long as you wrote it to be robust enough to accept a single file. 

The way this is currently set up - if you had 3 gene lists, you'd have to run the script twice to concatenate all of the lists.

**Pull request merged**